### ISSUE TITLE
Execution failed for task ':app:mergeExtDexDebug'.

### ISSUE THREAD

**Issue by Sarkheelmughal** [liodali/osm_flutter#562]
[√] Flutter (Channel stable, 3.24.1, on Microsoft Windows [Version 10.0.22631.3958], locale en-US)
    • Flutter version 3.24.1 on channel stable at F:\mobDevTools\flutterSDK\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 5874a72aa4 (4 weeks ago), 2024-08-20 16:46:00 -0500
    • Engine revision c9b9d5780d
    • Dart version 3.5.1
    • DevTools version 2.37.2
    
    i m using the latest version of flutter sdk and flutter_osm_plugin but i got the below issues. Then i tried flutter_osm_plugin ^0.70.3 then i was able to install my app
    
file:///C:/Users/HomePC/AppData/Local/Pub/Cache/hosted/pub.dev/flutter_osm_plugin-1.3.3/android/src/main/kotlin/hamza/dali/flutter_osm_plugin/utilities/ExtensionOSM.kt:197:21 Not enough information to infer type variable T

FAILURE: Build completed with 2 failures.

1: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':app:mergeExtDexDebug'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Failed to transform appcompat-resources-1.7.0.aar (androidx.appcompat:appcompat-resources:1.7.0) to match attributes {artifactType=android-dex, asm-transformed-variant=NONE, dexing-enable-desugaring=true, dexing-enable-jacoco-instrumentation=false, dexing-is-debuggable=true, dexing-min-sdk=23, org.gradle.category=library, org.gradle.dependency.bundling=external, org.gradle.libraryelements=aar, org.gradle.status=release, org.gradle.usage=java-runtime}.
      > Execution failed for DexingWithClasspathTransform: C:\Users\HomePC\.gradle\caches\transforms-3\152ab6d342da7c146f1c8cafb8ce67db\transformed\jetified-appcompat-resources-1.7.0-runtime.jar.
         > Error while dexing.
   > Failed to transform appcompat-1.7.0.aar (androidx.appcompat:appcompat:1.7.0) to match attributes {artifactType=android-dex, asm-transformed-variant=NONE, dexing-enable-desugaring=true, dexing-enable-jacoco-instrumentation=false, dexing-is-debuggable=true, dexing-min-sdk=23, org.gradle.category=library, org.gradle.dependency.bundling=external, org.gradle.libraryelements=aar, org.gradle.status=release, org.gradle.usage=java-runtime}.
      > Execution failed for DexingWithClasspathTransform: C:\Users\HomePC\.gradle\caches\transforms-3\a7c646a1bc579b036755820f23842f4c\transformed\appcompat-1.7.0-runtime.jar.
         > Error while dexing.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
==============================================================================

2: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':flutter_osm_plugin:compileDebugKotlin'.
> A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
   > Compilation error. See log for more details

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
==============================================================================

* Get more help at https://help.gradle.org

BUILD FAILED in 1m 15s
Error: Gradle task assembleDebug failed with exit code 1

**Comment by liodali** [comment]
did you try to upgrade gradle version ?

**Comment by lvenier** [comment]
I had a similar issues and had to : 
- add lines into android/app/build.gradle : 
```
android {
    compileSdkVersion flutter.compileSdkVersion
    ndkVersion flutter.ndkVersion
    namespace "a.b.c"
    compileOptions {
        coreLibraryDesugaringEnabled true
        sourceCompatibility JavaVersion.VERSION_1_8
        targetCompatibility JavaVersion.VERSION_1_8
    }
dependencies {
    coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:2.1.2'
```
- updagrade gradle in android/build.gradle : 
```
classpath 'com.android.tools.build:gradle:8.4.0'
```
Hope it helps

**Comment by liodali** [comment]
i will hotfix tonight

**Comment by liodali** [comment]
new version already published `1.3.3+1`

**Comment by Sarkheelmughal** [comment]
Thanks for quick response .
This version fixed that issue 

distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-all.zip
